openedx/features/qverse_features/registration/helpers.py

```diff
@@ -23,8 +23,8 @@ def get_file_header_row(file_content, encoding):
     io_string = io.StringIO(decoded_file)
     dialect = Sniffer().sniff(io_string.readline())
     io_string.seek(0)
-    header_row = reader(io_string, delimiter=dialect.delimiter).next()
-    return [heading.lower().strip() for heading in header_row]
+    dict_reader = reader(io_string, delimiter=dialect.delimiter)
+    return [row for row in dict_reader]
```

Comment:
Just asking, can we do this?
```suggestion
    return list(reader(io_string, delimiter=dialect.delimiter))
```